fedimint-core/Cargo.toml

```diff
@@ -60,15 +60,12 @@ url = { workspace = true, features = ["serde"] }
 
 [dev-dependencies]
 test-log = { workspace = true }
-tokio-test = "0.4.4"
```

Comment:
I guess this one's still needed, check the CI failing step.

Reply:
Fixed!